docs/faqs.md

```diff
@@ -438,3 +438,23 @@ Generally no, these sorts of options can most likely be set from a profile or do
 and would otherwise dynamically embed this into the CMake config files or generated pkg-config files then it should be allowed.
 
 Doing so requires [deleting the option from the `package_id`](adding_packages/conanfile_attributes.md#removing-from-package_id).
+
+## Can I use full_package_mode for a requirement in my recipe?
+
+For some unusual projects, they need to be aligned and being used as requirement, using the very same version, options and settings.
+Those projects usually break between patch versions and are very sentsitive, so we can not use different versions throught Conan graph dependencies,
+otherwise, it may result on unexpected behavior, or even runtime errors.
+
+A very kwnon project is [glib](https://conan.io/center/glib), which requires the very same configuration to prevent multiple instances when using static link. As solution, we could consume glib on full package id mode, like:
+
+```python
+def package_id(self):
+    self.info.requires["glib"].full_package_mode()
+```
+
+Perfect solution on consumer side, but there is a painful side-effect: CCI will not re-generate all involved packages for any change in the dependencies graph which glib is associated, which means, users will start to see **MISSING_PACKAGES** error during their pull requests.
```

Comment:
that's something to explore for a future, as with embed mode in v2 problem will appear way more often.
possible solution:
- a new tapaholes-like background job to re-generate downstream graph
- allow `--build missing` or `--build xxx` or similar in certain PRs
- allow to build non-standard configurations (e.g. via special keywords in PR title or first message), like overriding default options, versions of dependencies and so on
- expose an interface to build missing (or non-standard) packages on demand (via GH comment, or dedicated web-page)